doc/modules/model_evaluation.rst

```diff
@@ -2,9 +2,9 @@
 
 .. _model_evaluation:
```

Comment:
Should we change the ref as well?

Reply:
I wish we could but this will break existing URLs